### PR TITLE
test(route): testing route group inside route group

### PR DESCRIPTION
**Pull request by andresilvagomez**
## Proposed changes

Testing for multiples route groups!

## Types of changes

What types of changes does your code introduce?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/adonisjs/http-server/blob/master/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by andresilvagomez** [comment]
@thetutlage  @RomainLanz

**Comment by thetutlage** [comment]
May I how is this a breaking change?

**Comment by andresilvagomez** [comment]
@thetutlage sorry yes I updated it!

File: test/group.spec.ts [hunk 1]
```diff
@@ -341,4 +341,70 @@ test.group('Route Group', () => {
       },
     ])
   })
+
+  test('multiples routes group insede of group', (assert) => {
```
Comment (thetutlage) [inline test/group.spec.ts]: There is typo in `inside` spelling.

Comment (RomainLanz) [inline test/group.spec.ts]:
```suggestion
  test('multiples routes group inside of group', (assert) => {
```

**Pull request closed without merging**